options_test.go

```diff
@@ -0,0 +1,40 @@
+package envbuilder_test
+
+import (
+	"testing"
+
+	"github.com/coder/envbuilder"
+	"github.com/coder/serpent"
+	"github.com/stretchr/testify/require"
+)
+
+type testEnvCase struct {
+	Env      string
+	EnvValue string
+	Expected interface{}
+	Actual   func() interface{}
+}
+
+func TestEnvOptions(t *testing.T) {
+	t.Setenv("SETUP_SCRIPT", "setup.sh")
+	t.Setenv("CACHE_TTL_DAYS", "7")
+	t.Setenv("IGNORE_PATHS", "/var,/tmp")
+	t.Setenv("SKIP_REBUILD", "true")
+	t.Setenv("GIT_CLONE_SINGLE_BRANCH", "")
+
+	var o = envbuilder.Options{}
+	cmd := serpent.Command{
+		Options: o.CLI(),
+		Handler: func(inv *serpent.Invocation) error {
+			return nil
+		},
+	}
+	err := cmd.Invoke().WithOS().Run()
+	require.NoError(t, err)
+
+	require.Equal(t, o.SetupScript, "setup.sh")
+	require.Equal(t, o.CacheTTLDays, int64(7))
```

Comment:
I'd suggest choosing a large number to validate that the current code is handling integers as 64-bit

Reply:
Hmm.. we currently use just lower numbers, to be honest. In this case, using `int64` is a `coder/serpent` thing.